app/src/main/java/com/github/sdp/ratemyepfl/database/ItemsRepository.kt

```diff
@@ -4,33 +4,40 @@ import com.github.sdp.ratemyepfl.model.items.Classroom
 import com.github.sdp.ratemyepfl.model.items.Classroom.Companion.toClassroom
 import com.github.sdp.ratemyepfl.model.items.Course
 import com.github.sdp.ratemyepfl.model.items.Course.Companion.toCourse
+import com.github.sdp.ratemyepfl.model.items.Restaurant
+import com.github.sdp.ratemyepfl.model.items.Restaurant.Companion.toRestaurant
+import kotlinx.coroutines.tasks.await
 import javax.inject.Inject
 
-class ItemsRepository @Inject constructor() : ItemsRepositoryInterface, Repository() {
+class ItemsRepository @Inject constructor(): ItemsRepositoryInterface, Repository() {
     private val collectionCourses = db.collection("courses")
     private val collectionRooms = db.collection("rooms")
+    private val collectionRestaurants = db.collection("restaurants")
 
     companion object {
         private const val TAG_COURSES = "CourseRepository"
         private const val TAG_ROOMS = "ClassroomRepository"
+        private const val TAG_RESTAURANTS= "RestaurantRepository"
     }
 
-    override suspend fun getCourses(): List<Course?> {
-        return getLimit(collectionCourses, 50)
-            .mapNotNull { obj -> obj.toCourse() }
+    override suspend fun getCourses() : List<Course?> {
```

Comment:
Why did you change this part, I think Julien refactored some part to avoid duplication

Reply:
Merging error, my bad :(